thoth/user_api/api_v1.py

```diff
@@ -571,6 +571,22 @@ def list_buildlogs(page: int = 0):
     return _do_listing(BuildLogsStore, page)
 
 
+def get_python_package_versions_count():
+    """Retrieve number of Python package versions in Thoth Knowledge Graph."""
+    from .openapi_server import GRAPH
+    return GRAPH.get_python_package_versions_count_all(
+    )
+
+
+def get_python_package_versions_count_per_package_name(page: int = 0):
+    """Retrieve number of Python package versions per package name in Thoth Knowledge Graph."""
+    from .openapi_server import GRAPH
+    return GRAPH.get_python_package_versions_all_count(
+        count=PAGINATION_SIZE,
+        start_offset=page*PAGINATION_SIZE
+    )
```

Comment:
It would be great if we return pagination info in response headers.

Reply:
```
 x-page: 0 
 x-page-size: 100 
```